tests/workflows/integration/test-upload-charm/test.Dockerfile

```diff
@@ -0,0 +1 @@
+FROM ubuntu:latest
```

Comment:
Can we reuse the already existing OCI image to make sure eveything is connected as expected?

Reply:
Do you mean using a pre-existed dockerfile in the test dummy charm?

Reply:
Yes, if you think is worth. It is just a suggestion